services/cinema_hall.py

```diff
@@ -0,0 +1,19 @@
+from db.models import CinemaHall

```

Comment:
Please add a blank line between different types of imports and change the import ordering to first import third-party libraries.